.travis.yml

```diff
@@ -29,6 +32,12 @@ script:
   - docker ps -a 
   - docker exec client python ./test/functional/setup/setup_dbs.py -dbname $SQLSRV_DBNAME
   - docker exec client python ./test/functional/setup/setup_dbs.py -dbname $PDOSQLSRV_DBNAME
+  - docker exec client cp ./source/shared/msodbcsql.h ./test/functional/setup/
```

Comment:
does msodbcsql.h has to be in the setup folder the compiling and building msKSP.so?